db/data/movies.js

```diff
@@ -0,0 +1,19 @@
+const apiCalls = require("../../apiCalls.js")
+
+const movieData = async (genre) => {
+  const moviesByGenre = await apiCalls.movieFetch(genre)
+  let moviesToPost = []
+  let counter = 0;
+  while (moviesToPost.length < 25) {
+    let randomMovieTitle = moviesByGenre[counter].split('/')[2]
+    let randomMovieDetails = await apiCalls.singleMovie(randomMovieTitle)
+    if (randomMovieDetails) {
+      moviesToPost.push(randomMovieDetails)
+    }
+    counter++
+    setTimeout(() => {console.log(`${counter} / 25`)}, 250)
```

Comment:
The free version of this API limits calls for every one second I think, so this is to avoid that...

Reply:
Good call!

Reply:
That is so sneaky!!, I have never heard of that for a free API. You are giving me more reasons to make sure to look closer to the  API's documentation.